packages/components/src/inputs/docs/passwordField/description.tsx

```diff
@@ -0,0 +1,10 @@
+import { PasswordField, Label, HelperMessage } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <PasswordField placeholder="Placeholder">
+            <Label>Password:</Label>
+            <HelperMessage>Helper message</HelperMessage>
```

Comment:
I would change this for a hint such as : more than 10 characters.

Reply:
true, I did passwordField first and forgot to go back and put "real life" text